### ISSUE TITLE
Ethernet Bridge

### ISSUE THREAD

**Issue by Kryptonit3** [FalconChristmas/fpp#1642]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Most people who use a RPi with FPP for a projector connect it with WiFi and to their projector. It would be nice to natively have a tab on the networking page to allow the Ethernet port (eth0) be used as a bridge to allow network control of the projector.

I have found a few pages with info on how to do this. I am attempting to see if it is feasible by manually implementing.

https://www.willhaley.com/blog/raspberry-pi-wifi-ethernet-bridge/
https://www.raspberrypi.com/documentation/computers/configuration.html#use-your-raspberry-pi-as-a-network-bridge
https://github.com/arpitjindal97/raspbian-recipes

The bridge.sh script gave me an error the first time I ran it so I needed to run `sudo apt-get install dhcpcd5` first. Can’t figure out how to get the projector to get an IP.

**Comment by bloojoop** [comment]
Just a comment, software bridging can become quite cpu intensive.  I would be mindful of the cpu load when considering software bridging.

**Comment by Kryptonit3** [comment]
> Just a comment, software bridging can become quite cpu intensive. I would be mindful of the cpu load when considering software bridging.

I agree, but this specific case the most that it will do is sent shutdown or power up commands. The traffic will be extremely low and not constant.

**Comment by darylc** [comment]
@Kryptonit3 One problem is that whilst you may not be intending to do much traffic over a bridge, other users would try to use it to bridge ethernet/wifi for bulk traffic etc.  Personally I'm not in favour of turning FPP into a bridge.

**Comment by darylc** [comment]
The FPP developers discussed this and decided to close this issue.
